.gitignore

```diff
@@ -22,3 +22,7 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+firebase_config
```

Comment:
do you think that `firebase_config` can be move out of the `.gitignore` since the `apiKey` and other firebase variables is not hard coded anymore and moved to a .env file? Also I think it would be best to create a .env template file with instructions for the adding a user database configs it. I have seen this in other repos where a `.env` file is needed.  here is a link to an article on how to create one: [Best Practices Around Committing .env Files To Version Control](https://www.getfishtank.com/blog/best-practices-for-committing-env-files-to-version-control)